pkg/upstream/types/types.go

```diff
@@ -34,8 +34,13 @@ type Upstream struct {
 }
 
 type Update struct {
-	Cursor       string `json:"cursor"`
-	VersionLabel string `json:"versionLabel"`
+	ChannelID    string     `json:"channelID"`
+	ChannelName  string     `json:"channelName"`
+	Cursor       string     `json:"cursor"`
+	VersionLabel string     `json:"versionLabel"`
+	ReleaseNotes string     `json:"releaseNotes"`
+	ReleasedAt   *time.Time `json:"releasedAt"`
```

Comment:
`omitempty` for pointer types?

Reply:
this struct isn't returned to the client anywhere, just internal api usage